pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1291,11 +1350,15 @@ func (t *Task) deleteRsyncResources() error {
 		return err
 	}
 
+	t.Log.Info(fmt.Sprintf("Checking for stale Rsync resources on source MigCluster [%v/%v]",
+		t.PlanResources.SrcMigCluster.Namespace, t.PlanResources.SrcMigCluster.Name))
 	err = t.findAndDeleteResources(srcClient)
 	if err != nil {
 		return err
 	}
 
+	t.Log.Info(fmt.Sprintf("Checking for stale Rsync resources on destination MigCluster [%v/%v]",
```

Comment:
For my own edification, what makes a resource stale?

Reply:
This is a sort of generic term for resources that are either:

1) Leftover from a previous migration where we exited unexpectedly and weren't able to clean up
2) Leftover from a migration that ran successfully, and the resources (rsync pods, secrets, etc) that are no longer needed are considered "stale"

Reply:
So we can never adopt a resource that is pre-created? 

that is fine, just making sure I understand :)

Reply:
I think anything that we are considering "stale" is something we explicitly don't want to leave sitting around for some reason, but I get your point that we should re-use resources where possible if there aren't resource issues associated with doing so.